app/javascript/utils/index.ts

```diff
@@ -73,8 +73,7 @@ export const clickTracking = (affiliate: string, module: string, query: string,
       module_code: module,
       position,
       query,
-      vertical,
-      type: 'click'
+      vertical
```

Comment:
`type` was added in error to #1488 and is resulting in some un-permitted params errors -- the type is actually added later in the click tracking pipeline and can be removed here.